testsrc/org/mozilla/javascript/tests/Test262SuiteTest.java

```diff
@@ -436,6 +438,52 @@ public static void tearDownClass() {
     private final Test262Case testCase;
     private final boolean markedAsFailing;
 
+    /** @see https://github.com/tc39/test262/blob/main/INTERPRETING.md#host-defined-functions */
+    public class $262 {
+        private ScriptableObject scope;
+
+        public $262() {}
+
+        public $262(ScriptableObject scope) {
+            this.scope = scope;
+
+            scope.put("$262", scope, this);
+            scope.setAttributes("$262", ScriptableObject.DONTENUM);
```

Comment:
I think this should be done in the setup and mot in the ctor.

Reply:
With `setup` you mean into the buildScope method?

That would entail duplicating the code between there and the $262.createRealm method, which i don't really like either

Reply:
Ok, is see, my problem was as i saw this line

    new $262(scope);

i was really confused because i did not expect that this ctor call does some kind of registration in the scope. So from the first look this line looks really useless.

Maybe a factory method that has a hint in the name about the registration?

Reply:
How about now?